website/docs/tutorials/02-installation.md

```diff
@@ -32,6 +32,13 @@ To upgrade Protostar, run:
 $ protostar upgrade
 ```
 
+# Protostar Versioning
```

Comment:
Why do we actually include it in docs? Maybe the better place is wiki?

Reply:
agree

Reply:
although it doesn't hurt to include it here as well.

Reply:
It is a unnecessary clutter IMO but i won't block it or anything, it doesn't hurt

Reply:
A user needs to decide whether to upgrade Protostar or not. At the beginning, we bumped PATCH every 2 weeks. 

https://docs.expo.dev/workflow/expo-go/#sdk-versions

Reply:
I find this section to be much more developer-centric than user-centric. What I think users could find relatable is a statement which says "we release every 2 weeks; consult release notes for information about any breaking changes"

Reply:
As an anegdote, I am not familiar with release structure of any tool I am using :p, you guys are? Why would I need this information. I just go to release notes if something breaks and check what has changed

Reply:
A user needs to make the decision whether it's a good moment for updating Protostar or not. `cairo-lang` and Protostar aren't super stable.

Reply:
I don't think this uscase is well thought. User can just upgrade protostar, if everything works he is happy. Otherwise they can downgrade protostar (we support it in installation scripts). I think you are overestimating avg dev thought process

Reply:
I think it is better, not sure about the others

Reply:
Naturally I agree.